code/modules/events/ghost_role/space_dragon.dm

```diff
@@ -1,7 +1,7 @@
 /datum/round_event_control/space_dragon
 	name = "Spawn Space Dragon"
 	typepath = /datum/round_event/ghost_role/space_dragon
-	weight = 7
+	weight = 5
```

Comment:
this is config isn't it?

Reply:
so yaknow, talk to moth about this

Reply:
this isnt dynamic either but these are basically completely unused outside of greenshifts

Reply:
oh it... oh god you're right